scss/6-components/_callout.scss

```diff
@@ -22,94 +22,93 @@
     margin-bottom: 0;
   }
 
-  &.cads-callout- {
-    &example {
-      border: solid $cads-border-width-small $cads-language__border-colour;
-      border-radius: $cads-border-radius;
-      border-left-color: $cads-palette__mid-grey;
-      border-left-width: $cads-border-width-large;
-
-      h2:first-of-type,
-      h3:first-of-type {
-        &::before {
-          margin-bottom: $cads-spacing-3;
-          color: $cads-language__secondary-text-colour;
-          font-size: 0.875rem;
-          line-height: 0.875rem;
-          display: block;
-
-          @include cads-media-breakpoint-up(md) {
-            font-size: 1rem;
-          }
-
-          content: 'Example';
-        }
+  &,
+  &.cads-callout-standard {
+    border-color: $cads-language__brand-secondary;
+    background-color: cads-colour-to-bg($cads-language__brand-secondary);
+  }
 
-        .cads-lang-cy &::before {
-          content: 'Example';
+  &.cads-callout-example {
```

Comment:
the background color of this yellow, think it should be white?

but what's really weird is that the regression tests are passing with a 0 %diff:
![image](https://user-images.githubusercontent.com/2331893/95445454-f52aea80-0956-11eb-8bad-c72116593840.png)

Reply:
Ah interesting, yeah that was a mistake, let me double check that.

Reply:
Fixed that now. I think this was because these are _new_ snapshots, as we've migrated them. But corrected this and set an explicit background on the example callout.

Reply:
FAB.

The backstop thing is weirding me out tho - if the reference image and the test image are different it shouldn't pass, even if the image is new.